WBooks/BookDetailViewController.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  BookDetailViewController.swift
+//  WBooks
+//
+//  Created by Matías David Schwalb on 30/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+final class BookDetailViewController: UIViewController {
+
+    private let bookID: Int
+    private let bookDetailView: BookDetailView = BookDetailView.loadFromNib()!
+    private let bookDetailController = BookDetailController()
+    private var bookDetailViewModel = BookDetailViewModel(bookID: -1)
+    
+    let dispatchGroup = DispatchGroup()
+    
+    init(bookID: Int) {
+        self.bookID = bookID-1
+        self.bookDetailViewModel.bookID = bookID
+        super.init(nibName: "BookDetailViewController", bundle: Bundle.main)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        self.bookID = -1
+        super.init(coder: aDecoder)
+    }
+
+    override func loadView() {
+        view = bookDetailView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupNav()
+        
+        bookDetailController.bookDetail.rentButton.addTapGestureRecognizer { _ in
+            print("Rent Button tapped")
+            self.rent()
+        }
+        
+        bookDetailController.bookDetail.addToWishlistButton.addTapGestureRecognizer { _ in
+            print("Add to wishlist button tapped")
+        }
+    }
+    
+    func rent() {
+        var rentStatus: Bool = true
+        if checkBookStatus() {
+            
+            dispatchGroup.notify(queue: .main) {
+                rentStatus = self.requestRent()
+            }
+            
+            if rentStatus {
+                rentRequestSuccessful()
+            } else {
+                rentRequestFailed()
+            }
+        } else {
+            bookIsUnavailable()
+        }
+        
+    }
+    
+    func checkBookStatus() -> Bool {
+        return BookDB.bookArrayDB[bookID].status == "available"
+    }
+    
+    func requestRent() -> Bool {
```

Comment:
Let's move this method into the View Model too.

Reply:
applied these changes to newer branch (comment-development) rather than this one. Should we cancel this one and merge comment-development?